.github/workflows/ci.yml

```diff
@@ -0,0 +1,58 @@
+name: ci
```

Comment:
```suggestion
name: CI
```
Can we have it in capital letters?

Reply:
Yes, I changed. Thank you

Firstly I changed name to 'CI'. But then was needed file renaming. And I tryed lowercase for both. But seems, it is still need uppercase to make GHA badge works.

Reply:
Actually, we don't need the filename to match the name of the workflow. It was perfectly fine with the `ruby.yml` file and `name: CI`.

Reply:
Thank you. I corrected filename.